packages/snap/src/rpc/substrate/sendUnit.ts

```diff
@@ -0,0 +1,19 @@
+import ApiPromise from "@polkadot/api/promise";
+import {Wallet} from "../../interfaces";
+import { Keyring } from "@polkadot/api";
+import { Hash } from '@polkadot/types/interfaces/runtime';
+
+export async function sendUnit(wallet: Wallet, api: ApiPromise, amount: string|number, to: string): Promise<Hash> {
+  const keyring = new Keyring();
+  const keypair = keyring.addFromJson(wallet.getPluginState().polkadot.account.keyring);
+  const { nonce } = await api.query.system.account(keypair.address);
```

Comment:
https://polkadot.js.org/api/start/api.tx.html

According to this, sign and send will obtain next nonce by itself